.github/workflows/ci.yml

```diff
@@ -69,6 +69,40 @@ jobs:
           name: unit-test-report
           path: payments-core/build/reports/tests/testDebugUnitTest/
 
+
```

Comment:
I have this hear to quickly see the build results, but I plan to move it to instrumentations_tests.yml before I merge this PR.